nixos/modules/services/networking/cgproxy.nix

```diff
@@ -0,0 +1,110 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.cgproxy;
+  settingsFormat = pkgs.formats.json { };
+  settingsFile = settingsFormat.generate "config.json" cfg.settings;
+in
+{
+
+  options = {
+    services.cgproxy = {
+      enable = mkEnableOption (lib.mdDoc "cgproxy");
+
+      settings = mkOption {
+        description = lib.mdDoc ''
+          options priority:
```

Comment:
A link to some documentation is missing otherwise we have no idea how to use this. Also can we link to some doc explaining the priority instead?